### ISSUE TITLE
Create an API required field text should be at the top of the form

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#11133]
### Description:

![image](https://user-images.githubusercontent.com/20179540/116360111-331d2c80-a81d-11eb-974c-cf79d13af185.png)

like in
![image](https://user-images.githubusercontent.com/20179540/116360027-1ed92f80-a81d-11eb-8448-499a2542e55f.png)

### Environment details (with versions):
- OS:
- Client:
- Env (Docker/K8s):

---
